set.mm

```diff
@@ -11820,11 +11821,11 @@ This definition (in the form of ~ dfifp2 ) appears in Section II.24 of
   norasslem3 $p |- ( -. ph -> ( ( ps -> ch ) <-> ( ( ph \/ ps ) -> ch ) ) ) $=
     ( wn wo biorf imbi1d ) ADBABECABFG $.
 
-  $( A characterization of when an expression involving nor associates.  This
-     is identical to the case when alternative denial is associative, see
-     ~ nanass .  Remark:  Like alternative denial, nor is also commutative, see
-     ~ norcom .  (Contributed by RP, 29-Oct-2023.)  (Proof shortened by Wolf
-     Lammen, 17-Dec-2023.) $)
+  $( A characterization of when an expression involving joint denials
+     associates.  This is identical to the case when alternative denial is
+     associative, see ~ nanass .  Remark:  Like alternative denial, nor is also
```

Comment:
Then "nor" should be replaced here, too (by "joint denial").

Maybe "(joint denial)" and "(alternative denial)" should be added to the section headers "Logical "nor"" and "Logical "nand" (Sheffer strole)", and "(logical "nor")" and "(logical "nand") to the section headers "Joint denial" and "Alternative denial".

Reply:
Thanks, I'll do it in my upcoming PR. I modified it to be consistent with nanass. As for the other places, maybe we should indeed add them, to have both.

"NAND" and "NOR" are like nicknames for "alternative denial" and "joint denial" respectively, but they are widely used, so I don't know if we should prefer one or the other (in titles and their definition, we should provide both). In intuitionistic logic, NOR is indeed "joint denial", but NAND is weaker than "alternative denial". More faithful names would be non-conjunction and non-disjunction.